x/leverage/keeper/oracle.go

```diff
@@ -0,0 +1,60 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+
+	"github.com/umee-network/umee/x/leverage/types"
+)
+
+// Price returns the USD value of an sdk.Coin containing base assets
+func (k Keeper) Price(ctx sdk.Context, coin sdk.Coin) (sdk.Dec, error) {
```

Comment:
Maybe GetUSDValue?

Reply:
Currently it's been changed from `GetValue/GetTotalValue` to `Price/TotalPrice`, so that's probably how it'll stay for now.